src/ProjectTemplates/Shared/TemplatePackageInstaller.cs

```diff
@@ -80,7 +80,7 @@ public static async Task<ProcessEx> RunDotNetNew(ITestOutputHelper output, strin
                 DotNetMuxer.MuxerPathOrDefault(),
                 //--debug:disable-sdk-templates means, don't include C:\Program Files\dotnet\templates, aka. what comes with SDK, so we don't need to uninstall
                 //--debug:custom-hive means, don't install templates on CI/developer machine, instead create new temporary instance
-                $"new {arguments} --debug:disable-sdk-templates --debug:custom-hive \"{CustomHivePath}\"");
+                $"new {arguments} --force --debug:disable-sdk-templates --debug:custom-hive \"{CustomHivePath}\"");
```

Comment:
If this can't be combined w/ `Project.RunDotNetNewAsync(...)` (which has a better name😀), please use the same approach to handling retries in both.

Reply:
I don't need this change anymore, I added the delete everything if directory exists instead which seems safer, I'll remove this change